### PR TITLE
book: General minor improvements

### PR DESCRIPTION
**Pull request by BourgondAries**
This patch attempts to improve sentence structure, grammar, and flow.

### CODE REVIEW

File: docs/md_book/src/intro.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ and its source code is completely yours to tinker with. The code is available on
 [am]: https://github.com/amethyst/laminar/tree/master
 
 ## Motivation
-Laminar is fully written in Rust and therefore it has no garbage collector, no data-races, and memory safety. 
+Laminar is fully written in Rust and therefore has no garbage collector, no data-races, and is completely memory safety.
```
Comment (TimonPost) [inline docs/md_book/src/intro.md]: `completely memory safety.`

File: docs/md_book/src/intro.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ and its source code is completely yours to tinker with. The code is available on
...
 That's why laminar is a good candidate to be a safe and better replacement for other reliable-UDP implementations.
-This library is written for use in the Amethyst game engine, however, we fully believe that this library can become an excellent reliable UDP implementation in its own right.
+This library is originally written for use in the Amethyst game engine, however, laminar can operate fully without Amethyst.
```
Comment (TimonPost) [inline docs/md_book/src/intro.md]:
```suggestion
This library is originally written for use in the Amethyst game engine, however, Laminar can operate fully without Amethyst.
```

File: docs/md_book/src/intro.md (hunk 1 above)
Comment (jstnlef) [reply docs/md_book/src/intro.md]: I'm also not entirely sure of the "is completely memory safe". It isn't exactly true.

File: docs/md_book/src/intro.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 ## Introduction
 
-Welcome! This book will teach you everything you need to know about of the networking in laminar.
+Welcome! This book will teach you everything you need to know about networking in laminar.
```
Comment (jstnlef) [inline docs/md_book/src/intro.md]: Should probably be "networking with Laminar`

File: docs/md_book/src/intro.md (hunk 1 above)
Comment (BourgondAries) [reply docs/md_book/src/intro.md]: @jstnlef Rust without `unsafe` is completely memory safe. We should disallow unsafe code via the crate lint before asserting this as a fact about laminar.

**Pull request merged**